keel-api/src/main/kotlin/com/netflix/spinnaker/keel/rest/ExportController.kt

```diff
@@ -55,7 +67,12 @@ class ExportController(
     @RequestParam("serviceAccount") serviceAccount: String
   ): SubmittedResource<*> {
     val apiVersion = SPINNAKER_API_V1.subApi(cloudProviderOverrides[cloudProvider] ?: cloudProvider)
-    val kind = CaseFormat.LOWER_CAMEL.to(CaseFormat.LOWER_HYPHEN, type)
+    val singularType = if (type.endsWith(char = 's', ignoreCase = true)) {
```

Comment:
Would it be less sketchy to just include plurals in the `typeToKind`?

Reply:
Aye, that def looks less sketch